stage/westeurope/gateway/application/terragrunt.hcl

```diff
@@ -0,0 +1,118 @@
+# Include all settings from the root terragrunt.hcl file
+include {
+  path = find_in_parent_folders()
+}
+
+terraform {
+  source = "git::git@github.com:pagopa/io-infrastructure-modules-new.git//azurerm_application_gateway"
+}
+
+dependency "resource_group" {
+  config_path = "../resource_group"
+}
+dependency "log_analytics_workspace" {
+  config_path = "../../common/log_analytics_workspace"
+}
+dependency "vnet" {
+  config_path = "../../common/virtual_network"
+}
+dependency "key_vault" {
+  config_path = "../../common/key_vault"
+}
+
+inputs = {
+  name                  = "gateway"
+  resource_group_name   = dependency.resource_group.outputs.resource_name
+
+  # Subnet
+  virtual_network_name       = dependency.vnet.outputs.resource_name
+  subnet_address_prefix      = "10.0.1.0/24"
+  subnet_resource_group_name = dependency.vnet.outputs.resource_group_name
+
+  # IP
+  ip_sku               = "Standard"
+  ip_allocation_method = "Static"
+
+  # Application Gateway 
+  key_vault_id               = dependency.key_vault.outputs.id
+  log_analytics_workspace_id = dependency.log_analytics_workspace.outputs.id
+  certificate_name           = "certs-STAGE-IO-ITALIA-IT-DATA"
+  certificate_password       = "certs-STAGE-IO-ITALIA-IT-PASSWORD"
+
+  services = [
+    {
+      hl = {
+        name                       = "listener-api-443"
+        host_name                  = "api.stage.io.italia.it"
+        protocol                   = "Https"
+        require_sni                = true
+        custom_error_configuration = {}
+      }
+      pb = {
+        name                = "probe-api"
+        interval            = 30
+        protocol            = "Http"
+        path                = "/status-0123456789abcdef"
```

Comment:
Is this a real path or just for testing?

Reply:
/status-0123456789abcdef is the real path that must be used to probe Azure APIM